### PR TITLE
Minor typo in Korean docs fixed Fix 01

### PR DESCRIPTION
**Pull request by NitishKumar06**
Fix: #34909

Note: This PR actually contains the solution of the 1st change requested. The 2nd change lies in this PR:https://github.com/kubernetes/website/pull/35062

### CODE REVIEW

File: content/ko/docs/reference/access-authn-authz/authorization.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ DELETE    | delete(개별 리소스), deletecollection(리소스 모음)
 
 쿠버네티스 API 서버는 몇 가지 인가 모드 중 하나를 사용하여 요청을 승인할 수 있다.
 
- * **Node** - 실행되도록 스케줄된 파드에 따라 kubelet에게 권한을 부여하는 특수 목적 인가 모드. Node 인가 모드 사용에 대한 자세한 내용은 [Node 인가](/docs/reference/access-authn-authz/node/)을 참조한다.
+ * **Node** - 실행되도록 스케줄된 파드에 따라 kubelet에게 권한을 부여하는 특수 목적 인가 모드. Node 인가 모드 사용에 대한 자세한 내용은 [Node 인가를](/docs/reference/access-authn-authz/node/) 참조한다.
```
Comment (yoonian) [inline content/ko/docs/reference/access-authn-authz/authorization.md:90]:
```suggestion
 * **Node** - 실행되도록 스케줄된 파드에 따라 kubelet에게 권한을 부여하는 특수 목적 인가 모드. Node 인가 모드 사용에 대한 자세한 내용은 [Node 인가](/docs/reference/access-authn-authz/node/)를 참조한다.
```

Comment (yoonian) [reply content/ko/docs/reference/access-authn-authz/authorization.md:90]: @NitishKumar06 I have a correction for you already mentioned in the last PR. I encourage you to apply. The Korean term '를' is a postposition, not part of 'Node 인가', node authentication.

**Comment by NitishKumar06** [comment]
Definitely I'll follow the same by today. Thank you!

**Comment by NitishKumar06** [comment]
Hey @yoonian Please look at the changes now!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Pull request merged**